### ISSUE TITLE
Activation range breaks chicken farms

### ISSUE THREAD

**Issue by dsampinski** [Wesley1808/ServerCore#33]
When chickens are out of range (16), they stop laying eggs for the farm to work, even though ticking every second is enabled.

ServerCore: 1.3.0
Minecraft: 1.19

**Comment by Wesley1808** [comment]
The time for a chicken to lay an egg is counted in ticks - and if it ticks less often, it will take that much longer for it to finish.
I can probably work around it by updating the egg time as part of its inactive tick. It shouldn't be completely broken though, but it does take 20 times longer for a chicken to lay an egg this way.

**Comment by Wesley1808** [comment]
I did end up finding a few more 'somewhat' similar cases to mobs that are affected by it.
Now in most cases this makes sense logically, otherwise it isn't much "activation range".

In case of chicken eggs, I feel like thats just harmless enough to fix.
The other thing I managed to find was about skeleton horse traps. 
They would normally despawn after 18.000 ticks, but that obviously doesn't happen if that counter doesn't get updated if nobody is close to them..
